tools/profiler.py

```diff
@@ -84,6 +84,18 @@ def forward(self, *args, **kwargs):
         return self.model(*args, **kwargs)
 
 
+class IpuWrapper(torch.nn.Module):
```

Comment:
There should be no `IpuWrapper`. If you want to count the elapse of IPU forward, just do it in `mmdeploy/backend/ipu/wrapper.py`

Reply:
when I do the latency test with profiler.py, without this 'IpuWrapper', profile with ipu would error out

Reply:
What's the error? Just put the decorator `@TimeCounter.count_time(Backend.IPU.value)` to the forward of `IPUWrapper`.